content/Certification/certifying_a_product/development.md

```diff
@@ -0,0 +1,201 @@
+
+Test DACs are not permitted during certification testing, and devices must arrive with two instances with individually provisioned DACs in order to show that the device can meet the requirements around individual provisioning. Devices at certification are not required to use production DACs.
+
+Device manufacturers may wish to consider joining the CSA early in the process to understand the details of the security procedures within the Alliance.
+
+### Certification Declaration (CD)
+
+The Certification Declaration (CD) is provided by the CSA after a device is
+certified. It is tied to the vendor and product ID (or IDs) of the certified
+device, and is signed by CSA. The public key corresponding to the signing key is
+well known and distributed by the CSA through the DCL.
+
+- [CD Signing Certs Root CA](https://on.dcl.csa-iot.org/dcl/pki/all-certificates?subjectKeyId=97:E4:69:D0:C5:04:14:C2:6F:C7:01:F7:7E:94:77:39:09:8D:F6:A5)
+
+- [CD Signing Cert SKIDs](https://on.dcl.csa-iot.org/dcl/pki/child-certificates/MFIxDDAKBgNVBAoMA0NTQTEsMCoGA1UEAwwjTWF0dGVyIENlcnRpZmljYXRpb24gYW5kIFRlc3RpbmcgQ0ExFDASBgorBgEEAYKifAIBDARDNUEw/97:E4:69:D0:C5:04:14:C2:6F:C7:01:F7:7E:94:77:39:09:8D:F6:A5)
+
+The CD is NOT a standard x.509 certificate, but is
+instead a CMS-encoded SignedData payload containing a TLV-encoded structure,
+described in section 6.3.1 Certification Declaration of the spec. Certification
+Declarations can be viewed using the
+[Certificate Tool](https://project-chip.github.io/connectedhomeip-doc/src/tools/chip-cert/README.html),
+
+The certification declaration for the product line is provided after
+certification testing, and therefore cannot be hard coded in the firmware. New
+certification declarations are issued for each certified firmware update.
+Therefore, product development should include a strategy for initial
+provisioning and updates of the certification declaration.
+
+See
+[Preparing a Device for Certification](#preparing-a-device-for-certification)
+for discussion of CDs used for certification testing.
+
+### Matter on-boarding materials (QR codes and manual codes)
+
+In addition to DACs, each individual unit needs to be provisioned with its own
+on-boarding material for discovery and initial commissioning. This includes the
+following per-unit items provisioned on the device:
+
+-   discriminator
+-   PAKE verifier (encodes the passcode and PAKE salt)
+-   PAKE salt
+
+It also includes the following items encoded on the device exterior or
+packaging, both of which encode the discriminator and passcode for the
+corresponding unit:
+
+-   QR code
+-   manual code
+
+Manual codes are required for all devices, QR codes are optional but
+recommended. The rules for QR and manual code inclusions are covered in section
+5.7.6 and section 13.6 of the specification.
+
+[Matter Setup Code Guidelines](https://groups.csa-iot.org/wg/members-all/document/27481)
```

Comment:
```suggestion
```

All the content in the doc above is also contained in the brand guidelines below